tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+STARCCM_INSTALLATION_TIMEOUT = 1800
+STARCCM_JOB_TIMEOUT = 600
+STARCCM_LICENCE_SECRET = "perf_integ_secret"
+TASK_VCPUS = 36  # vCPUs are cut in a half because multithreading is disabled
```

Comment:
Improvement (for the next iteration): this value should be dynamically retrieved from EC2, according to the instance type that the test is going to use, i.e. c5n.18xl in this case.